content/docs/dvclive/api-reference/index.md

```diff
@@ -35,22 +35,13 @@ Including `save_dvc_exp=True` will automatically
 ### Log data
 
 <toggle>
-<tab title="Parameters">
+<tab title="Artifacts">
```

Comment:
I don't think we should put it before metrics.

Reply:
Should we add it to the example further down the page?

Reply:
Also need to add it to api-reference/live#methods

Reply:
No strong preference, I just updated to alphabetical order to pick one order

Reply:
Do you think we should also add to get started?

Reply:
Okay, it looks like `Images` are out of order then 😄 ?